### PR TITLE
Report compressed/uncompressed size of icudtl, snapshots, and libflutter

### PR DESCRIPTION
**Pull request by goderbauer**
Reported for the hello_world compile benchmark in devicelab for now.

/cc @yjbanov

### CODE REVIEW

File: dev/devicelab/lib/tasks/perf_tests.dart [hunk 1]
```diff
@@ -349,6 +358,42 @@ class CompileTest {
...
+
+  static Future<Map<String, dynamic>> getSizesFromApk(String apkPath) async {
+    final  String output = await eval('unzip', <String>['-v', apkPath]);
+    final List<String> lines = output.split('\n');
+
+    final Map<String, _UnzipListEntry> fileToMetadata = <String, _UnzipListEntry>{};
+
+    // First three lines are header, last two lines are footer.
+    for (int i = 3; i < lines.length - 2; i++) {
+      final List<String> line = lines[i].trim().split(new RegExp('\\s+'));
```
Comment (yjbanov) [inline dev/devicelab/lib/tasks/perf_tests.dart]: nit: the parsing logic is split between this loop and the `_UnzipListEntry` constructor. I'd move everything into the constructor.

File: dev/devicelab/lib/tasks/perf_tests.dart [hunk 2]
```diff
@@ -349,6 +358,42 @@ class CompileTest {
...
+    }
+
+    final _UnzipListEntry icudtl = fileToMetadata['assets/icudtl.dat'];
+    final _UnzipListEntry libflutter = fileToMetadata['lib/armeabi-v7a/libflutter.so'];
+    final _UnzipListEntry isolateSnapshotData = fileToMetadata['assets/isolate_snapshot_data'];
+    final _UnzipListEntry isolateSnapshotInstr = fileToMetadata['assets/isolate_snapshot_instr'];
+    final _UnzipListEntry vmSnapshotData = fileToMetadata['assets/vm_snapshot_data'];
+    final _UnzipListEntry vmSnapshotInstr = fileToMetadata['assets/vm_snapshot_instr'];
+
+    return <String, dynamic>{
```
Comment (yjbanov) [inline dev/devicelab/lib/tasks/perf_tests.dart]: I wonder if the dashboard UI is ready to onboard so many new metrics? :smile:

**Review by yjbanov — APPROVED** [review]

**Comment by yjbanov** [comment]
lgtm

File: dev/devicelab/lib/tasks/perf_tests.dart (hunk 2 above)
Comment (goderbauer) [reply dev/devicelab/lib/tasks/perf_tests.dart]: Hehe. I actually cleaned up the dashboard on Friday and removed all deprecated (dart1) benchmarks.

File: dev/devicelab/lib/tasks/perf_tests.dart (hunk 1 above)
Comment (goderbauer) [reply dev/devicelab/lib/tasks/perf_tests.dart]: Good call! Done.

**Pull request merged**